ddp-workspace/projects/ddp-pancan/src/app/components/anchors-page/anchors-page.component.ts

```diff
@@ -9,5 +10,18 @@ import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
 export class AnchorsPageComponent {
     @Input() source: string;
     @Input() route: string;
+    @Input() linksMap?: { [key: string]: string };
+
     anchorLabel = 'section';
+    readonly examplesUrl = [
+        'https://www.nature.com/articles/s41467-020-19291-x',
+        'https://www.annalsofoncology.org/article/S0923-7534(19)40698-4/fulltext',
+        'https://www.nature.com/articles/s41591-019-0749-z.epdf?author_access_token=cBBx7BAXQ6Z3D-FtgKtMatRgN0jAjWel9jnR3ZoTv0OuyqpOiBwU8Z8ASb5NY9IbXk1V1KF_R5v6TTbcRFo-JeMJ0-VLX-DiOKjqfH1Idg9OXZftog_CeSSuTvzZX78hU5-xKsQ6XRMok9B76DOedQ%3D%3D'
+    ];
```

Comment:
I'm not sure it's a good idea to just rely on the order in json file, that data source is totally out of the component. I'd recommend either moving urls to json file or handle each example separately by the unique key w/o loop.

Reply:
Fixed.